### ISSUE TITLE
Uncaught TypeError: 'caller', 'callee', and 'arguments' properties may not be accessed on strict ...

### ISSUE THREAD

**Issue by johnhaley81** [Axosoft/atom-github-notifications#1]
This is what I did from the point of installing the plugin.

1. Add the plugin to Atom
2. Generate key
3. Set `Mark as read` option in settings
4. Went through and dismissed all of my notifications that came up
5. Start and update of other plugins that had updates.
6. Blurred the window.

**Atom**: 1.14.4 x64
**Electron**: 1.3.13
**OS**: Mac OS X 10.12.3
**Thrown From**: [atom-github-notifications](https://github.com/axosoft/atom-github-notifications) package 0.1.1

### Stack Trace

Uncaught TypeError: 'caller', 'callee', and 'arguments' properties may not be accessed on strict mode functions or the arguments objects for calls to them

```
At /Users/johnh/.atom/packages/atom-github-notifications/node_modules/github-xhr/dist/error.js:104

TypeError: 'caller', 'callee', and 'arguments' properties may not be accessed on strict mode functions or the arguments objects for calls to them
    at /packages/atom-github-notifications/node_modules/github-xhr/dist/error.js:104:71)
    at XMLHttpRequest.ontimeout (/packages/atom-github-notifications/node_modules/github-xhr/dist/index.js:657:22)
```

### Commands

```
     -7:59.1.0 settings-view:open (input.hidden-input)
     -7:54.3.0 editor:delete-to-beginning-of-line (input.hidden-input)
     -7:50.5.0 core:confirm (input.hidden-input)
     -6:47.2.0 editor:delete-to-beginning-of-line (input.hidden-input)
     -6:42.9.0 core:confirm (input.hidden-input)
     -4:34.3.0 core:paste (input.hidden-input)
     -1:13.5.0 settings-view:check-for-package-updates (atom-workspace.workspace.scrollbars-visible-always.seti-icons.seti-roboto.seti-animate.theme-base16-tomorrow-dark-theme.theme-seti-ui.find-visible)
```

### Non-Core Packages

```
.bin undefined 
advanced-open-file 0.16.5 
atom-beautify 0.29.17 
atom-github-notifications 0.1.1 
autocomplete-clang 0.10.0 
dash 1.7.1 
docblockr 0.9.1 
editorconfig 2.2.2 
fonts 2.0.0 
git-time-machine 1.5.6 
highlight-selected 0.12.0 
hyperclick 0.0.40 
js-hyperclick 1.10.1 
language-babel 2.55.6 
language-gitignore 0.3.0 
linter 1.11.23 
linter-eslint 8.1.2 
merge-conflicts 1.4.4 
minimap 4.26.8 
minimap-find-and-replace 4.5.1 
minimap-highlight-selected 4.5.0 
minimap-pigments 0.2.2 
minimap-selection 4.4.0 
pigments 0.39.0 
regex-railroad-diagram 0.16.0 
seti-ui 1.6.1 
sort-lines 0.14.0 
Sublime-Style-Column-Selection 1.7.3 
tree-ignore 0.6.2 
vim-mode-plus 0.82.3 
vim-mode-plus-keymaps-for-surround 0.2.1 
wordcount 2.10.4 
```

**Comment by cbargren** [comment]
Looks like an issue with `github-xhr` referencing `arguments.callee` in strict mode. This'll be fixed shortly.

**Comment by cbargren** [comment]
This should be fixed with `0.1.7`, please reopen if you see it again though. 👍
